### PR TITLE
Fix code comment

### PR DESCRIPTION
**Pull request by codebien**
~Add the Sum stat into the resolvers map. It is helpful for reusing the same function from the Remote Write output. https://github.com/grafana/xk6-output-prometheus-remote/pull/64~

Just fix a comment

### CODE REVIEW

File: metrics/value_type.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ import "errors"
 // Possible values for ValueType.
 const (
 	Default = ValueType(iota) // Values are presented as-is
-	Time                      // Values are timestamps (nanoseconds)
+	Time                      // Values are timestamps (milliseconds)
```
Comment (na--) [inline metrics/value_type.go:8]:
```suggestion
	Time                      // Values are durations (milliseconds)
```

**Review by na-- — COMMENTED** [review]
Hmm :thinking: if I am reading this correctly: https://github.com/grafana/k6/blob/616e5e2f24ba41995f4d8bf9e1fab8f89285fb44/cmd/options.go#L191-L200

Then changing this function will also allow users to specify `k6 run --summary-trend-stats min,med,max,sum`, which is not something we want, right? :thinking: 

Why do you need this change?

**Comment by na--** [comment]
I am not sure how the sum of a `Trend`'s values would be useful in any way for the end-of-test summary? Maybe it could be of use in some weird `handleSummary()` use case? :thinking: Even so, AFAIK, nobody has requested something like this so far.

Also, the `sum` is easy to get with our current `TrendSink` implementation, since it doesn't require any calculation - we update it on every value `add()` call. However, if we adopt HDR histograms (https://github.com/grafana/k6/issues/763), calculating the sum would [require some computation](https://github.com/openhistogram/circonusllhist/blob/c78ce013c914a89b519b0e9e184e62589cb727a9/circonusllhist.go#L771-L784) and even so, it will be just an approximation.

So, given the dubious value to users and the potential complication for HDR histograms, I think copying some code to the extension (with a `TODO`) would be preferable to hastily updating the values in this k6 function.

**Comment by codebien** [comment]
Ok, then I think I can do something hacky for excluding the `sum` from the validation and still reuse the k6 function. Thanks for your opinion :bow:

**Comment by na--** [comment]
The fix to the `metrics.Time` comments were good though :sweat_smile:

**Review by oleiade — APPROVED** [review]

**Pull request merged**